OJP/OJP_Common.xsd

```diff
@@ -449,7 +449,21 @@
 					<xs:documentation>Internal code of the attribute. Can be used for detection of double occurrences.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:group ref="AllFacilitiesGroup"/>
+			<xs:element name="Key" type="xs:NMTOKEN" minOccurs="0">
```

Comment:
Do you want to enforce some Identity-Key-Constraint on this property so it is unique?

Reply:
I guess it is not necessary unique in a file. E.g. the same Key/Value might occur in every TripResult. So, I think we leave it at that.

Reply:
It should work like the NeTEx keyValue list. Hence, per TripResult only one time this key may exists.

Reply:
I don't think so. E. we have a special dining car and we provide that information for two legs but not the third, then the information would be on the TimedLeg/LegAttribute  with the same Key and the same Value within the same Trip.
So I don't think we can do a hard constraint.